### PR TITLE
Add Random Color Node

### PR DESCRIPTION
**Pull request by kdsin23**
Add Random Color Node

### CODE REVIEW

**Review by OmarEmaraDev — CHANGES_REQUESTED** [review]
Looks good. But remember to update the CHANGELOG and add the node to the menu.

**Comment by kdsin23** [comment]
> Looks good. But remember to update the CHANGELOG and add the node to the menu.

Thanks, i will add it soon

**Comment by kdsin23** [comment]
why i updated the CHNGELOG.md, then appeared ''This branch has conflicts that must be resolved"? how to solve?

**Comment by OmarEmaraDev** [comment]
Merge upstream master like we said before, then add the CHANGELOG entry.

File: animation_nodes/nodes/color/random_color.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import bpy
+import random
+from bpy.props import *
+from ... data_structures import Color
+from ... events import propertyChanged
+from ... base_types import AnimationNode
+from . c_utils import generateRandomColors
+
+class RandomColor(bpy.types.Node, AnimationNode):
+    bl_idname = "an_RandomColor"
```
Comment (OmarEmaraDev) [inline animation_nodes/nodes/color/random_color.py:10]: `an_RandomColor` -> `an_RandomColorNode`

**Review by OmarEmaraDev — CHANGES_REQUESTED** [review]

File: animation_nodes/nodes/color/random_color.py (hunk 1 above)
Comment (kdsin23) [reply animation_nodes/nodes/color/random_color.py:10]: Ok!

**Review by OmarEmaraDev — APPROVED** [review]
Thanks!

**Pull request merged**